glue/CollectorLanguageInterfaceImpl.cpp

```diff
@@ -206,10 +206,57 @@ MM_CollectorLanguageInterfaceImpl::scavenger_internalGarbageCollect_shouldPercol
 }
 
 GC_ObjectScanner *
-MM_CollectorLanguageInterfaceImpl::parallelScavenger_getObjectScanner(MM_EnvironmentStandard *env, omrobjectptr_t objectPtr, void *allocSpace, uintptr_t flags)
+MM_CollectorLanguageInterfaceImpl::scavenger_getObjectScanner(MM_EnvironmentStandard *env, omrobjectptr_t objectPtr, void *allocSpace, uintptr_t flags)
```

Comment:
Same comments as the example glue.  This is the base for all new languages so it definitely needs to be properly \#if defined and have all code in the right order